openlibrary/templates/type/author/view.html

```diff
@@ -152,16 +152,15 @@ <h2 class="collapse">
                         <a href="$changequery(sort=None)#editions">Most Editions</a> |  <a href="$changequery(sort='old')#editions">First Published</a> | <a href="$changequery(sort='new')#editions">Most Recent</a> | <strong class="lightgreen">Title</strong>
                 </span>
 
-                <span class="mode-options">
-                  <input type="radio" name="mode" value="everything" id="mode-search-everything" class="search-mode">
-                  <label for="mode-search-everything">Everything</label>
-                  <input type="radio" name="mode" value="ebooks" id="mode-search-ebooks" class="search-mode">
-                  <label for="mode-search-ebooks">Ebooks</label>
-                  <input type="radio" name="mode" value="printdisabled" id="mode-search-printdisabled" class="search-mode">
-                  <label for="mode-search-printdisabled">Print Disabled</label>
-                </span>
-</p>
-
+                <p>
+                  $if input(mode="everything").mode == "everything":
```

Comment:
It looks like mode print-disabled is no longer an option? Or am I missing something?

Reply:
Yes, you're right, but the mechanism for enabling this was/is sufficiently broken that it seemed worth removing until we have a more comprehensive solution. Ebooks only will also show print -disabled results (there's not currently a way to filter them out)